pegasus/sites.v3/code.org/public/student.haml

```diff
@@ -0,0 +1,74 @@
+:css
+
+  .full-resource-block {
+    width: 100%;
+    margin-bottom: 30px;
+    height: 244px;
+  }
+  .description {
+    padding-bottom: 10px;
+    color: #fff;
+  }
+  .resource-title {
+    padding: 0px;
+    margin: 0px;
+    font-size: 24px;
+    color: #FFFFFF;
+    padding-bottom: 10px;
+  }
+  .side-image {
+    width: 485px;
+    height: 244px;
+    float: left;
+  }
+  .side-image.right, .side-description-area.right {
+    border-top-right-radius: 5px;
+    border-bottom-right-radius: 5px;
+  }
+  .side-image.left, .side-description-area.left {
+    border-top-left-radius: 5px;
+    border-bottom-left-radius: 5px;
+  }
+  .side-description-area {
+    float: left;
+    width: 485px;
+    height: 100%;
+    background-color: #00adbc;
+    padding: 30px 30px;
+  }
+  .gray-button {
+    background-color: #c6cacd;
+    border-color: #c6cacd;
+    color: #5b6770;
+  }
+
+.full-resource-block
```

Comment:
Can we add a little more margin between the header and the first block? See uk.code.org/student.